easybuild/easyconfigs/b/bx-python/bx-python-0.8.4-foss-2019a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'bx-python'
+version = '0.8.4'
+
+github_account = 'bxlab'
+homepage = 'https://github.com/%(github_account)s/%(name)s'
```

Comment:
It's usually preferable to keep the homepage literal so one can easy copy&paste the URL.
And since you're not using github_account anywhere else it is just wasteful.

Reply:
Fixed :+1: